grails-webhooks/grails-app/services/webhooks/WebhookService.groovy

```diff
@@ -87,6 +96,33 @@ class WebhookService {
         return plugin.onEvent(context,data) ?: new DefaultWebhookResponder()
     }
 
+    Long handleWebhookEventsData(EventQueryType queryType, RdWebhook webhook){
```

Comment:
I don't see the point on having "EventQueryType" if it's just delete. Also the method name should be something related to what is doing.
Also i think there's no need to return the number of the records deleted, instead it should be logged here and just be void.
Please add javadoc to the method

Reply:
done